lib/constants.ts

```diff
@@ -13,3 +13,12 @@ export const FONT_SIZE_2 = '1.6rem'
 export const FONT_SIZE_3 = '1.4rem'
 export const FONT_SIZE_4 = '1.3rem'
 export const FONT_SIZE_5 = '1.2rem'
+
+export const POST_TITLE_COLOR = '#2f3234'
+export const POST_DESCRIPTION_COLOR = '#95989a'
+export const POST_BOX_SHADOW = '0 2px 6px rgba(0, 0, 0, 0.16)'
+export const POST_BORDER_RADIUS = '10px'
+export const POST_THUMBNAIL_HEIGHT = '165px'
+export const POST_THUMBNAIL = (src: string) => `url(${src}) no-repeat center`
```

Comment:
This is not constant anymore and it should not be in this file. We can create `utils.ts` file for these use cases.